### PR TITLE
Useless easter egg feature

### PR DESCRIPTION
**Pull request by lasgra**
Title is self explanatory

### CODE REVIEW

File: .github/workflows/deploy-dev.yml
Comment (lasgra) [inline .github/workflows/deploy-dev.yml:1]: Literally haven't changed anything in this file

File: .github/workflows/deploy-prod.yml
Comment (lasgra) [inline .github/workflows/deploy-prod.yml:1]: Same here

File: src/CustomValidators.ts
Comment (lasgra) [inline src/CustomValidators.ts:1]: Same here

Comment (lasgra) [inline src/CustomValidators.ts:1]: 💀

File: .github/workflows/deploy-dev.yml
Comment (Majkoo) [reply .github/workflows/deploy-dev.yml:1]: branch base you worked on is main.

this is PR to develop

git checkout develop | git fetch | git pull origin develop

File: .github/workflows/deploy-prod.yml
Comment (Majkoo) [reply .github/workflows/deploy-prod.yml:1]: same here

File: src/app/app.component.html [hunk 1]
```diff
@@ -1 +1,2 @@
 <router-outlet />
+<pp-easter-egg></pp-easter-egg>
```
Comment (Majkoo) [inline src/app/app.component.html]: `<pp-easter-egg/>`

File: src/app/shared/components/easter-egg/easter-egg.component.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  showEasterEgg(){
+    if (this.easterEgg != null){
+      this.easterEgg.nativeElement.style.display = "block"
+      this.easterEgg.nativeElement.style.left = (Math.random() * window.innerWidth).toString() + "px"
+      this.easterEgg.nativeElement.style.top = (Math.random() * window.innerHeight).toString() + "px"
+    }
+  }
+
+  clickedEgg(){
+    this.easterEgg.nativeElement.style.display = "none"
```
Comment (Majkoo) [inline src/app/shared/components/easter-egg/easter-egg.component.ts]: use native angular API to modify classes. Especially if we have Tailwind in place. Typically you should not use @ViewChild for such cases unless youre refering to a custom component and its context is not modifiable via native angular api

File: src/app/shared/components/easter-egg/easter-egg.component.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+  ngAfterViewInit(){
+    if (Math.random()*1000 == 693){
+      this.showEasterEgg()
+    }
+  }
+
+  showEasterEgg(){
+    if (this.easterEgg != null){
+      this.easterEgg.nativeElement.style.display = "block"
```
Comment (Majkoo) [inline src/app/shared/components/easter-egg/easter-egg.component.ts:22]: please use nativa angular api to edit element references.

You should not use @Viewchild unless its a custom component reference that cannot be handled via angular api

File: package.json [hunk 4]
```diff
@@ -41,7 +41,7 @@
     "@angular-eslint/eslint-plugin": "16.1.0",
     "@angular-eslint/eslint-plugin-template": "16.1.0",
     "@angular-eslint/schematics": "16.1.0",
-    "@angular-eslint/template-parser": "16.1.0",
+    "@angular-eslint/template-parser": "^4.3.1",
```
Comment (Majkoo) [inline package.json]: why is that

File: src/app/app.module.ts [hunk 5]
```diff
@@ -12,39 +12,41 @@ import {NgOptimizedImage} from "@angular/common";
...
-      RouterModule,
-      BrowserModule,
-      BrowserAnimationsModule,
-      AppRoutingModule,
-      HttpClientModule,
-      ServiceWorkerModule.register('ngsw-worker.js', {enabled: environment.production}),
-      LayoutModule,
-      NgOptimizedImage,
-      UrlTransformModule,
+    declarations: [
```
Comment (Majkoo) [inline src/app/app.module.ts]: please revert that object prop reordering

File: package.json (hunk 4 above)
Comment (lasgra) [reply package.json]: bro, idk

File: src/app/app.module.ts (hunk 5 above)
Comment (lasgra) [reply src/app/app.module.ts]: 🤓🤓🤓

File: package.json (hunk 4 above)
Comment (Majkoo) [reply package.json]: then revert it

**Review by Majkoo — COMMENTED** [review]
git good

**Review by Majkoo — APPROVED** [review]

**Comment by lasgra** [comment]
99% of gamblers quit before hitting big

**Pull request merged**